renovate.json

```diff
@@ -15,6 +17,40 @@
     {
       "matchFileNames": ["frontend/testing/cypress/**"],
       "ignoreDeps": ["yarn"]
+    },
+    {
+      "matchFileNames": ["src/App/**"],
+      "additionalBranchPrefix": "app/",
+      "commitMessageSuffix": " (app)"
```

Comment:
This should give us separate PRs for everything inside `src/App/` - I think that makes sense since we will have completely separate dependency graphs here